src/components/landing/Body.js

```diff
@@ -9,16 +9,18 @@ import '../../styles/landing/body.css'
 export default class Body extends Component {
   render() {
     return (
-      <div className='landing-main'>
+      <div className="landing-main">
         <Overlay />
-        <section className='quotations'>
-          <div className='quotations-container'>
-            <Quotation quotation='I found an apple thanks to Dragonboard'
-              url='/images/gravity2.png'
-              name='Isaac Newton' />
-            <Quotation quotation='We installed Dragonboard, sales are now up by 15%'
-              url='images/how-to-draw-john-adams-john-adams_1_000000016809_5.png'
-              name='John Adamos, CEO, Adam Apples' />
+        <section className="quotations">
+          <div className="quotations-container">
+            <Quotation quotation="I found an apple thanks to Dragonboard"
+              url="/images/gravity2.png"
+              name="Isaac Newton" />
+            <Quotation
+              quotation="We installed Dragonboard, sales are now up by 15%"
+              url="images/how-to-draw-john-adams-john-adams_1_000000016809_5.png"
+              name="John Adamos, CEO, Adam Apples"
+            />
```

Comment:
HTML/JSX tags should be closed on the same line as their last attribute (especially odd here since it's not internally consistent with the `Quote`, above)